src/ResponsesView.jsx

```diff
@@ -71,9 +69,8 @@ ErrorView.defaultProps = {
 
 const ResponsesView = ({ responses }) => {
   const views = responses.map((response, idx) => {
-    if (!response.hasError)
-      return <TrackView track={response} idx={idx} key={response.id} />;
-    return <ErrorView error={response} idx={idx} key={response.id} />;
+    if (!response.hasError) return <TrackView track={response} idx={idx} />;
```

Comment:
Key field is required for list elements in collection. You will have a warning in dev console about that. Key values can be generated or combined keys.
Check:
- https://ru.reactjs.org/docs/lists-and-keys.html
- https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Now I generate keys using shortid.